app/models/miq_request_task/state_machine.rb

```diff
@@ -21,29 +21,15 @@ def signal(phase)
     begin
       send(phase)
     rescue => err
-      if [:finish, :provision_error].include?(phase.to_sym)
-        _log.error("[#{err.message}] encountered during [#{phase}]")
+      case phase
```

Comment:
```suggestion
      case phase.to_sym
```
do we want to keep the `to_sym` here?

Reply:
@kbrock Done!